static/test/javascripts/spec/common/commercial/build-page-targeting.spec.js

```diff
@@ -166,6 +166,7 @@ define([
             audienceScienceGateway.getSegments = function () {
                 return {};
             };
+            storage.local.set('gu.alreadyVisited', 0);
```

Comment:
If another test is leaving localStorage dirty, the ideal approach is to add an `afterEach` in that spec which cleans it up after each test.